src/usr/local/www/head.inc

```diff
@@ -77,7 +77,7 @@ if (!isset($config['system']['webgui']['dashboardcolumns'])) {
 	<script type="text/javascript">
 	//<![CDATA[
 	var events = events || [];
-	var newSeperator = false;
+	var newSeparator = false;
```

Comment:
This line was added because `newSeperator' is used in `firewall_rules.php` without it being initiated. The variable name contains a typo - it was missed in a previous commit which changed the original variable name. This line can be completely removed. Relevant commits:
https://gitlab.netgate.com/pfSense/pfSense/-/commit/c049c15e08a1ce4bb3b71c121dc5057f6be74a8f
https://gitlab.netgate.com/pfSense/pfSense/-/commit/36742b464a1b4449e52cbd0b539fece507a3b23e